mdsuite/calculators/angular_distribution_function.py

```diff
@@ -278,7 +289,14 @@ def _prepare_data_structure(self):
         start_index = 0
         stop_index = 0
         for species in self.args.species:
-            stop_index += self.experiment.species[species].n_particles
+            try:
+                if isinstance(self.args.atom_selection, dict):
```

Comment:
which type can the atom selection be if not dict?

Reply:
A slice,  I believe that is actually its default